### ISSUE TITLE
Add BTicino Dimmer switch

### ISSUE THREAD

**Issue by paolomarchiori** [madchicken/homebridge-zigbee-nt#82]
Hello,
I am using a cc2531 flashed with CC2531ZNP-Prod from CC2531_DEFAULT_20190608.zip and homebridge-zigbee-nt v1.1.1 (v1.1.2 and 2.0 won't load because of #78). I am trying to pair a Legrand "Dimmer switch with neutral" (BTicino dimmer-interruttore connesso Living light N4411C) which is supported by zigbee-herdsman-converters but somehow doesn't get recognized properly. Event tried shortening the distance (to 10cm) but nothing changed.

Zigbee: allowing new devices to join.
Accepting joining whitelisted device '0x0004740000a2101e'
[1/17/2021, 16:31:15] [ZigBee] Interview progress started for device 0x0004740000a2101e
[1/17/2021, 16:31:16] [ZigBee] Device announce: 0x0004740000a2101e (undefined -  Dimmer switch with neutral
)
[1/17/2021, 16:31:17] [ZigBee] Found ZigBee device:  Device {
  ID: 2,
  _type: 'Router',
  _ieeeAddr: '0x0004740000a2101e',
  _networkAddress: 25219,
  _manufacturerID: 4129,
  _endpoints: [
    Endpoint {
      ID: 1,
      profileID: 260,
      deviceID: 256,
      inputClusters: [Array],
      outputClusters: [Array],
      deviceNetworkAddress: 25219,
      deviceIeeeAddress: '0x0004740000a2101e',
      clusters: [Object],
      _binds: [],
      _configuredReportings: [],
      meta: {}
    },
    Endpoint {
      ID: 242,
      profileID: undefined,
      deviceID: undefined,
      inputClusters: [],
      outputClusters: [],
      deviceNetworkAddress: 25219,
      deviceIeeeAddress: '0x0004740000a2101e',
      clusters: {},
      _binds: [],
      _configuredReportings: [],
      meta: {}
    }
  ],
  _manufacturerName: undefined,
  _powerSource: undefined,
  _modelID: ' Dimmer switch with neutral\u0000\u0000\u0000\u0000',
  _applicationVersion: undefined,
  _stackVersion: undefined,
  _zclVersion: undefined,
  _hardwareVersion: undefined,
  _dateCode: undefined,
  _softwareBuildID: undefined,
  _interviewCompleted: false,
  _interviewing: true,
  meta: {},
  _lastSeen: 1610897476363,
  _linkquality: 0
}
[1/17/2021, 16:31:17] [ZigBee] Unrecognized device: 0x0004740000a2101e undefined Dimmer switch with neutral
Zigbee: disabling joining new devices.

(while I'm here: the device's manual says it comes out of the box in "switch" mode and needs to be set as dimmer - does anyone know if will this be possible once paired? thanks)

**Comment by madchicken** [comment]
This device is still not supported by the plugin. It is essentially a light with a brightness support. With version 2.0 of the plugin you will be able to configure this device by yourself through the configuration. Please wait some day until 2.0 is out

**Comment by madchicken** [comment]
v2.0.0 is now out. Can you try adding this to the plugin configuration?

```
            "devices": [{
                "manufacturer": "BTicino",
                "models": [
                  "L441C/N4411C/NT4411C"
                ],
                "services": [{
                     "type": "bulb",
                     "meta": {
                         "brightness": true
                     }
                }]
            }]
```

**Comment by paolomarchiori** [comment]
Installed 2.0.0 and added the devices stanza to the plugin configuration but unfortunately the pairing request doesn't show up anymore (not a range issue since zigbee2mqtt acknowledges the request and accepts it).

**Comment by madchicken** [comment]
Sorry, maybe the device already paired? Can you check though the Web UI? Also, make sure the permit join switch is turned on when you init the pairing process.
Attach logs of homebridge startup please

**Comment by madchicken** [comment]
Will be available in v2.0.2
